apps/entry/imports.py

```diff
@@ -115,6 +123,51 @@ def get_team_logos():
                 currentTeam = image_data["teamNumber"]
                 currentImage = image_data["encodedAvatar"]
             team_image.save()
+def get_match_data():
+    for event in Event.objects.all():
```

Comment:
This needs to be split up into smaller functions. Think about creating a method like
`get_match_data_event(event: Event)`  and then looping over the function call.